### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by sankleta**
Added more sections, reformatted.

Please check also through RStudio

### CODE REVIEW

File: man/map_identifiers.Rd [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+target_bioelement, target_datasource)}
+
+\description{
+Matches one or a group of given element ids with unknown element ids according to given filters.
+}
+
+\arguments{
+	\item{source_ids}{a single identifier or a character vector of identifiers.}
+	\item{source_bioelement}{character string: the type of source element. Possible values are \code{"gene"}, \code{"protein"}, \code{"transcript"} and \code{"genesymbol"}.}
+	\item{source_datasource}{character string: source database name. Possible values are \code{"ncbigene"}, \code{"ensembl"}, \code{"uniprot"}, \code{"refseq"}, or \code{"-"}. The value of \code{"-"} corresponds to any database currently known. See Examples for the details. }
```
Comment (nikolamueller) [inline man/map_identifiers.Rd]: Would be helpful to set "-" as default/fallback value

Comment (sankleta) [reply man/map_identifiers.Rd]: Yep, I think so too. I'll do it as a next task to not to mix documentation with code changes

File: man/map_identifiers.Rd [hunk 2]
```diff
@@ -0,0 +1,90 @@
+\name{mapIdentifiers}
+\alias{mapIdentifiers}
+\title{Documentation}
+\usage{mapIdentifiers(source_ids, source_bioelement, source_datasource, 
+target_bioelement, target_datasource)}
+
+\description{
+Matches one or a group of given element ids with unknown element ids according to given filters.
```
[the remark this answers is not in the document]
Comment (sankleta) [reply man/map_identifiers.Rd]: Changed a bit

File: man/map_identifiers.Rd [hunk 3]
```diff
@@ -18,17 +18,19 @@ Matches one or a group of given element ids with unknown element ids according t
 
 \details{
 
+	This function returns all mappings with a distance of 1…3 for a given (set of) input identifiers (source_ids) to identifiers of a given type: the target_bioelement (gene/protein/transcript/genesymbol).
+	
 Mapping is done within the range of distances: 
 \itemize{
-\item distance 1 -- there is a direct mapping path between source element and target element;
-\item distance 2 -- there is one or multiple mapping paths between source element and target element through one intermediate step, e.g. a gene connected to a protein through a transcript;
+\item distance 1 -- there is a direct mapping path between source_bioelement and target_bioelement;
```
Comment (sankleta) [inline man/map_identifiers.Rd]: Disagree with this change. source_bioelement and target_bioelement are input field names. If I provide source_bioelement = gene and target_bioelement = protein, this phrase means "there is a direct mapping between gene and protein". I used source element and target element to describe an entity that contains id, element type and database. Then the phrase can be interpreted as "there is a direct mapping between source id, type database and target id, type, database". Maybe the meaning of "element" here is not intuitively clear though

**Pull request merged**